python/nano/setup.py

```diff
@@ -126,7 +126,10 @@ def setup_package():
 
     pytorch_110_requires = ["torch==1.10.1",
                             "torchvision==0.11.2",
-                            "intel_extension_for_pytorch==1.10.100;platform_system=='Linux'"]
+                            "intel_extension_for_pytorch==1.10.100;platform_system=='Linux' and \
+                            python_full_version>='3.9.0'",
+                            "intel_extension_for_pytorch==1.11.0;platform_system=='Linux' and \
+                            python_full_version<'3.9.0'"]
```

Comment:
should we just drop this 1.11 and let ipex uninstalled for users with <3.9?

Reply:
Yeah, I will change to this.